chapter2/tests/t20/terraform/dev/ec2.tf

```diff
@@ -0,0 +1,37 @@
+module "ec2_instances" {
+  source        = "terraform-aws-modules/ec2-instance/aws"
+  count         = var.ec2_instances_count
+  name          = "${local.labels.ec2_instance}-${count.index + 1}"
+  ami           = var.ec2_ami_id
+  instance_type = var.ec2_instance_type
```

Comment:
you didn't format your tf code - make this a habit.

Reply:
> you didn't format your tf code - make this a habit.

@dmitry-mightydevops
it was wrong version
Done